app/role_data.go

```diff
@@ -222,8 +228,35 @@ func addRoles(management *config.ManagementContext) (string, error) {
 	}
 
 	if len(admin.PrincipalIDs) == 0 {
-		admin.PrincipalIDs = []string{"local://" + admin.Name}
-		management.Management.Users("").Update(admin)
+		var tries = 0
```

Comment:
since we are adding the local principal resolving to the create lifecycle, you _do not_ need to to it for the admin user in role_data.go anymore